samples/ChatbotExamplev2/Bots/PersonalizerChatbot.cs

```diff
@@ -75,12 +67,14 @@ public LuisBot(BotServices services, PersonalizerClient personalizerClient)
         /// <param name="cancellationToken">(Optional) A <see cref="CancellationToken"/> that can be used by other objects
         /// or threads to receive notice of cancellation.</param>
         /// <returns>A <see cref="Task"/> that represents the work queued to execute.</returns>
-        public async Task OnTurnAsync(ITurnContext turnContext, CancellationToken cancellationToken = default(CancellationToken))
+        public override async Task OnTurnAsync(ITurnContext turnContext, CancellationToken cancellationToken = default(CancellationToken))
         {
+            await base.OnTurnAsync(turnContext, cancellationToken);
+
             if (turnContext.Activity.Type == ActivityTypes.Message)
             {
                 // Check LUIS model
-                var recognizerResult = await _services.LuisServices[LuisKey].RecognizeAsync(turnContext, cancellationToken);
+                var recognizerResult = await _coffeeRecognizer.RecognizeAsync(turnContext, cancellationToken);
                 var topIntent = recognizerResult?.GetTopScoringIntent();
                 if (topIntent != null && topIntent.HasValue && topIntent.Value.intent != "None")
                 {
```

Comment:
Line 141 - Please add comment indicating this is about Personalizer sample and how it is related.

Reply:
added comment